GitUI/BranchTreePanel/RepoObjectsTree.cs

```diff
@@ -216,9 +208,8 @@ void CollectFromNodes(TreeNodeCollection nodes)
         private void BeforeDoubleClickExpandCollapse(object sender, CancelEventArgs e)
         {
             // If node is an inner node, and overrides OnDoubleClick, then disable expand/collapse
-            if (treeMain.SelectedNode?.Tag is Node node
-                && node.Nodes.Count > 0
-                && IsOverride(node.GetType().GetMethod("OnDoubleClick", BindingFlags.Instance | BindingFlags.NonPublic)))
+            if (treeMain.SelectedNode?.Tag is Node node && node.HasChildren()
+                && IsOverride(node.GetType().GetMethod(nameof(OnDoubleClick), BindingFlags.Instance | BindingFlags.NonPublic)))
```

Comment:
[readability] If we're wrapping then we should place each expression on its own line